src/theme/index.hbs

```diff
@@ -72,13 +78,13 @@
 
                 <!-- Mobile navigation buttons -->
                 {{#previous}}
-                    <a href="{{link}}" class="mobile-nav-chapters previous">
+                    <a rel="prev" href="{{link}}#content" class="mobile-nav-chapters previous">
```

Comment:
What is `#content` supposed to do? When I test it locally it doesn't seem to change anything.